sktime/base.py

```diff
@@ -89,10 +96,19 @@ def copy(self):
 class Estimator(_base_methods_mixin):
 
     def __init__(self, model=None):
-        self._model = model if model is not None else self._create_model()
+        # we only need to create a default model in case the subclassing Estimator provides the partial_fit interface.
+        if hasattr(self.__class__, 'partial_fit') and model is None:
```

Comment:
Perhaps its easier if the model is always just `None` in the beginning (except a model instance is explicitly provided). Why do we have to create it here?

Reply:
it has to be created for the partial_fit case explicitly (somewhere), because we do not have the fit call produces a new model instance logic.

Reply:
Right, makes sense!